From e827f20c32a772488827ea518b7a02a0e87b5c15 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 17:47:50 +0200
Subject: [PATCH 3/4] Telephony: Fix AAPT warnings

Fixes:
res/layout/telephony_disallowed_preference_screen.xml:33: warn:
generated id 'android:id/empty' for external package 'android'.
res/layout/pref_dialog_editphonenumber.xml:24: warn:
generated id 'android:id/message' for external package 'android'.
res/layout/pref_dialog_editpin.xml:25: warn:
generated id 'android:id/message' for external package 'android'.

Change-Id: Iee4c477a36d9883d50795a2b8a4a3023fda1ec48
---
 res/layout/pref_dialog_editphonenumber.xml            | 2 +-
 res/layout/pref_dialog_editpin.xml                    | 2 +-
 res/layout/telephony_disallowed_preference_screen.xml | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/layout/pref_dialog_editphonenumber.xml b/res/layout/pref_dialog_editphonenumber.xml
index cb1651628..09330f187 100644
--- a/res/layout/pref_dialog_editphonenumber.xml
+++ b/res/layout/pref_dialog_editphonenumber.xml
@@ -21,7 +21,7 @@
     android:orientation="vertical"
     android:padding="?android:attr/dialogPreferredPadding">
 
-    <TextView android:id="@+android:id/message"
+    <TextView android:id="@android:id/message"
         style="?android:attr/textAppearanceMedium"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
diff --git a/res/layout/pref_dialog_editpin.xml b/res/layout/pref_dialog_editpin.xml
index a27869075..94cdadfbd 100644
--- a/res/layout/pref_dialog_editpin.xml
+++ b/res/layout/pref_dialog_editpin.xml
@@ -22,7 +22,7 @@
     android:orientation="vertical"
     android:padding="?android:attr/dialogPreferredPadding">
 
-    <TextView android:id="@+android:id/message"
+    <TextView android:id="@android:id/message"
         style="?android:attr/textAppearanceSmall"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
diff --git a/res/layout/telephony_disallowed_preference_screen.xml b/res/layout/telephony_disallowed_preference_screen.xml
index 18b4d9491..129f1b779 100644
--- a/res/layout/telephony_disallowed_preference_screen.xml
+++ b/res/layout/telephony_disallowed_preference_screen.xml
@@ -30,7 +30,7 @@
                 android:cacheColorHint="@android:color/white"
                 android:fadingEdgeLength="16dip" />
 
-        <TextView android:id="@+android:id/empty"
+        <TextView android:id="@android:id/empty"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 android:gravity="center"
-- 
2.17.1

